master/internal/authz/obfuscate_test.go

```diff
@@ -0,0 +1,53 @@
+package authz
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/determined-ai/determined/proto/pkg/containerv1"
+	"github.com/determined-ai/determined/proto/pkg/devicev1"
+)
+
+func ObfuscateContainerDevice(t *testing.T, device *devicev1.Device) {
+	devBrand := device.Brand
+	devType := device.Type
+	err := ObfuscateDevice(device)
```

Comment:
I think we should delete this line since ```ObfuscateContainer``` should call ```ObfuscateDevice``` for us and this is part of what we should try and test

Reply:
You're right, removed!